src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final int INDEX_COLOR = 9;

```

Comment:
```suggestion
    private static final int NUMBER_OF_COLORS = 9;

```
this is not an index